src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -425,39 +401,16 @@ public async Task MaxConnectionsPerServer_WaitingConnectionsAreCancelable()
         [Fact]
         public async Task SendAsync_Cancel_CancellationTokenPropagates()
         {
-            TaskCompletionSource<bool> clientCanceled = new TaskCompletionSource<bool>();
-            await LoopbackServerFactory.CreateClientAndServerAsync(
-                async uri =>
-                {
-                    var cts = new CancellationTokenSource();
-                    cts.Cancel();
-
-                    using (HttpClient client = CreateHttpClient())
-                    {
-                        OperationCanceledException ex = null;
-                        try
-                        {
-                            await client.GetAsync(uri, cts.Token);
-                        }
-                        catch (OperationCanceledException e)
-                        {
-                            ex = e;
-                        }
-                        Assert.True(ex != null, "Expected OperationCancelledException, but no exception was thrown.");
+            var cts = new CancellationTokenSource();
+            cts.Cancel();
```

Comment:
Nit: if the identity of the CancellationToken isn't being checked, the test can just use `new CancellationToken(true)`.